VinylTests/Arbitrary.swift

```diff
@@ -11,9 +11,9 @@ import SwiftCheck
 
 /// Generates an array of lowercase alphabetic `Character`s.
 let lowerStringGen =
-Gen<Character>.fromElements(in: "a"..."z")
-    .proliferateNonEmpty
-    .map(String.init)
+    Gen<Character>.fromElements(in: "a"..."z")
+        .proliferateNonEmpty
+        .map(String.init)
```

Comment:
I get a compiler error here `ambiguous use of 'init'` which I can only resolve by changing to `.map() { String.init($0) }`.  Do you not get this error?

Error detail:
```
/Users/michael/Documents/Development/github/Vinyl/VinylTests/Arbitrary.swift:16:14: error: ambiguous use of 'init'
        .map(String.init)
             ^
Swift.String:32:12: note: found this candidate
    public init<T>(stringInterpolationSegment expr: T) where T : CustomStringConvertible
           ^
Swift.String:16:12: note: found this candidate
    public init<S>(_ characters: S) where S : Sequence, S.Element == Character
           ^
```